### DISCUSSION TITLE
Allow option of marking as a volunteer other adults 18+ in the home

### DISCUSSION THREAD

**Discussion by simply-emily** [CareTogether/CareTogetherCMS#186] [Ideas]
Conditionally require training if an 18+ other adult in a host home wishes to serve as a volunteer as well.

**Comment by LarsKemmann** [comment]
Development on this feature request is being tracked in #180.

**Reply by LarsKemmann** [reply]
This discussion thread is being merged into #137 - discussions should continue there.
